### PR TITLE
New content

### PR DESCRIPTION
**Pull request by mpenciak**
This PR adds the new 2023/969 eprint paper, and the new Lurk episode of the Zk Podcast (also fixes episode 280's title).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -132,7 +138,8 @@ Code implementations and explorations related to the Nova proof system, includin
 ### Podcast episodes
 
 - *Podcast*: [ZK-Podcast Episode 277: Nova and beyond with Srinath Setty](https://www.youtube.com/watch?v=hRza_k8i16s)
-- *Podcast*: [ZK-Podcast Episode 280: Episode 280: ProtoStar with Benedikt Buenz and Binyi Chen](https://zeroknowledge.fm/280-2/)
+- *Podcast*: [ZK-Podcast Episode 280: ProtoStar with Benedikt Buenz and Binyi Chen](https://zeroknowledge.fm/280-2/)
+- *Podcast*: [Zk-Podcast Episode 281: Exploring Lurk: a nNew Language for Recursive zk-SNARKs with Chhi'mèd Künzang and François Garillot](https://zeroknowledge.fm/281-2/)
```
Comment (huitseeker) [inline README.md:142]:
```suggestion
- *Podcast*: [Zk-Podcast Episode 281: Exploring Lurk: a new Language for Recursive zk-SNARKs with Chhi'mèd Künzang and François Garillot](https://zeroknowledge.fm/281-2/)
```

File: README.md [hunk 2]
```diff
@@ -132,7 +138,8 @@ Code implementations and explorations related to the Nova proof system, includin
 ### Podcast episodes
 
 - *Podcast*: [ZK-Podcast Episode 277: Nova and beyond with Srinath Setty](https://www.youtube.com/watch?v=hRza_k8i16s)
-- *Podcast*: [ZK-Podcast Episode 280: Episode 280: ProtoStar with Benedikt Buenz and Binyi Chen](https://zeroknowledge.fm/280-2/)
+- *Podcast*: [ZK-Podcast Episode 280: ProtoStar with Benedikt Buenz and Binyi Chen](https://zeroknowledge.fm/280-2/)
```
Comment (huitseeker) [inline README.md:141]: As we say Künzang later, we should do this:
```suggestion
- *Podcast*: [ZK-Podcast Episode 280: ProtoStar with Benedikt Bünz and Binyi Chen](https://zeroknowledge.fm/280-2/)
```

**Review by huitseeker — COMMENTED** [review]
Thanks a bunch! I left a couple of suggestions.

**Comment by mpenciak** [comment]
I agree with the suggestions!

**Review by huitseeker — APPROVED** [review]
Thank you so much!

**Pull request merged**